docs/technical/editor-integration.md

```diff
@@ -1,8 +1,8 @@
-# Editor Integration
+# "Open in External Editor" integration
```

Comment:
maybe 
* "Open External Editor" integration
or 
* "Open in [your external editor]" integration

Reply:
I kind of want to keep the `in` around because it matches up with the menu item in Desktop, but the latter option feels a bit unclear. I'll drop the `in` and see how it feels...